File: Day-10/sumOfFirstLastDigit.java
import java.util.*;
class sumOfFirstLastDigit
{
	public static void main (String[] args) throws java.lang.Exception
	{
		Scanner scan = new Scanner(System.in);
		int n;
		System.out.print(" Enter a Number: ");
		n = scan.nextInt();
		if (n >= 1 && n <= 9)
		    System.out.println(" The Sum of First and Last Digit is: "+ (n+n));
		else {
		    int sum = n % 10;
		    while(n > 0) {
		        n /= 10;
		        if(n >= 1 && n <= 9)
	            	sum += n;
		    }
		    System.out.println(" The Sum of First and Last Digit is: " + sum);
		}
	}
}

File: Day-10/sum_of_first_last_digit.c
#include <stdio.h>

void main() {
	int n, t;
	printf(" Enter a Number: ");
	scanf("%d", &n);
	if(n>=1 && n<=9)
	    printf("\n The Sum Of First and Last Digit is: %d\n", n+n);
	else
	{
	    int sum = n % 10;
	    while(n>0) {
            if (n >= 1 && n <= 9)
	            sum += n;
	        n /= 10;
	    }
	        printf("\n The Sum Of First and Last Digit is: %d\n", sum);
	}
}

